app/partials/event_booking_form.html

```diff
@@ -0,0 +1,110 @@
+            <div ng-if="editingSelf" class="school-wrap" school-dropdown selected-school-urn="school.schoolId">
+              {{schools}}      
+            </div>
+            <input ng-if="editingSelf" ng-hide="school.schoolId" ng-model="school.schoolOther" type="text" placeholder="School Other (if not listed above)">
+            
+            <span ng-if="targetUser.role!='STUDENT'">
+              <label for="job-title">
+                <strong>Job Title</strong>
+                <span class="ru-form-required"></span>
+              </label>
+              <input id="job-title" name="job-title" type="text" ng-model="additionalInformation.jobTitle" required>
+            </span>
+
+            <span ng-if="targetUser.role=='STUDENT'">
+              <label for="year-group">
+                <strong>School Year Group</strong>
+                <span class="ru-form-required"></span>
+              </label>
+              <select id="year-group" name="year-group" ng-model="additionalInformation.yearGroup" required>
+                                        <option value="9">Year 9</option>
+                                        <option value="10">Year 10</option>
+                                        <option value="11">Year 11</option>
+                                        <option value="12">Year 12</option>
+                                        <option value="13">Year 13</option>
+                                        <option value="TEACHER">N/A - Teacher</option>
+                                        <option value="OTHER">N/A - Other</option>
+              </select>
+
+              <strong ng-show="event.teacher && additionalInformation.yearGroup == 'TEACHER' && targetUser.role != 'TEACHER'"><a ui-sref="pages({id: 'contact_us_teacher'})" target="_blank">Click to upgrade to a teacher account</a> for free!</strong>
+            </span>
+
+            <label for="medical-reqs">
+              <strong>Dietary Requirements or Relevant Medical Conditions</strong>
+              <span aria-haspopup="true" class="icon-help has-tip" data-ot="For example, it is important for us to know if you have a severe allergy and/or carry an EpiPen, are prone to fainting, suffer from epilepsy..."></span>
+            </label>
+            <input id="medical-reqs" name="medical-reqs" type="text" ng-model="additionalInformation.medicalRequirements">
+
+            <label for="access-reqs">
+              <strong>Accessibility Requirements</strong>
+              <span aria-haspopup="true" class="icon-help has-tip" data-ot="For example, please let us know if you need wheelchair access, hearing loop or if we can help with any special adjustments."></span>
+            </label>
+            <input id="access-reqs" name="access-reqs" type="text" ng-model="additionalInformation.accessibilityRequirements">
+
+            <div ng-show="additionalInformation.yearGroup != 'TEACHER' && additionalInformation.yearGroup != 'OTHER'" class="row">
```

Comment:
The ordinary booking form requires emergency contact info for teachers, so why does the admin version one not when booking the same user on the same event?

Reply:
I believe users signed as an admin get the emergency contact details.